<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Eurobattle.net Client</string>
  </property>
  <property name="windowIcon">
   <iconset resource="xpam.qrc">
    <normaloff>:/favicon.ico</normaloff>:/favicon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedKingdom"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>500</width>
     <height>330</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>500</width>
     <height>350</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">#centralWidget {
	background-color: black;
}

#tabWidget {
	margin: 0px
}

#tab0 {
	background-image: url(:/eb_back_small.png);
}
#tab1 {
	background-image: url(:/eb_back_small.png);
}
#tab2 {
	background-image: url(:/eb_back_small.png);
}
#tab3 {
	background-image: url(:/eb_back_small.png);
}

QLabel {
	color: white;
}

QCheckBox {
	color:white;
}</string>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>501</width>
      <height>331</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab0">
     <attribute name="title">
      <string>Launcher</string>
     </attribute>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>220</y>
        <width>73</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Windowed</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>160</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Start without GProxy</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>100</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Start with GProxy</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab1">
     <attribute name="title">
      <string>GProxy console</string>
     </attribute>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>471</width>
        <height>221</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab2">
     <attribute name="title">
      <string>Update</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>90</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Get Beta patch</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowserUpdate">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>471</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="betapinbox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab3">
     <attribute name="title">
      <string>About</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>100</y>
        <width>181</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Eurobattle.net Client version 0.2.0.1</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>160</y>
        <width>341</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>We are searching for Qt/ C++ developers to further expand this client.</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>391</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="debugLabel">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>330</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="xpam.qrc"/>
 </resources>
 <connections/>
</ui>
